groundcontrol/admin-bundle.tasks.js

```diff
@@ -97,27 +104,23 @@ adminBundle.tasks.bundleOptimized = createBundleTask({
                     })
                 }
             ]
-        },
-        plugins: [
-            new webpack.optimize.UglifyJsPlugin({
-                mangle: true,
-                sourceMap: true,
-                output: {
-                    comments: false
-                }
-            })
-        ]
+        }
     },
     logStats: true
 });
 
 adminBundle.tasks.bundlePolyfills = createBundleTask({
     config: {
-        entry: ['babel-polyfill', adminBundle.config.srcPath + 'jsnext/polyfills.js'],
+        entry: ['babel-polyfill', `${adminBundle.config.srcPath}jsnext/polyfills.js`],
         output: {
-            filename: adminBundle.config.distPath + 'js/admin-bundle-polyfills.js',
+            filename: 'admin-bundle-polyfills.js',
+            path: path.resolve(__dirname, '../src/Kunstmaan/AdminBundle/Resources/public/js'),
```

Comment:
Can you use the adminBundle.config here? As all other paths are in the config object